### PR TITLE
Remove top/bottom padding on first/last items in a list

### LINKED ISSUE #2533: [Enhancement] Remove top/bottom padding on list with items
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement
On lists with items, the first item has 8px top padding and the last item has 8px bottom padding.

## Describe the solution you'd like
Remove first / last item padding.

## Have you considered any alternatives?
No!

## Are there any additional context?

List padding exploration

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

### PR DESCRIPTION
**Pull request by mark-drastrup**
## Which issue does this PR close?

This PR closes #2533

## What is the new behavior?
This first item in a `kirby-list` now has 0px `padding-top,` and the last item has 0px `padding-bottom`.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [x] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/src/lib/components/list/list.component.integration.spec.ts
Comment (RasmusKjeldgaard) [inline libs/designsystem/src/lib/components/list/list.component.integration.spec.ts]: Do you think these tests have any value now? If not we could remove them.

File: libs/designsystem/src/lib/components/item/item.component.scss
Comment (RasmusKjeldgaard) [inline libs/designsystem/src/lib/components/item/item.component.scss]: Could it be that the two other rules be obsolete now? They seem to reset the additional padding in a specific scenario, but now there is no additional padding.

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]

File: libs/designsystem/src/lib/components/item/item.component.scss
Comment (mark-drastrup) [reply libs/designsystem/src/lib/components/item/item.component.scss]: Yeah that could be. I'm gonna do a quick test to make sure we don't break anything :)

File: libs/designsystem/src/lib/components/list/list.component.integration.spec.ts
Comment (mark-drastrup) [reply libs/designsystem/src/lib/components/list/list.component.integration.spec.ts]: That's a good point! Let's remove them :)

File: libs/designsystem/src/lib/components/item/item.component.scss
Comment (mark-drastrup) [reply libs/designsystem/src/lib/components/item/item.component.scss]: You're right. We can remove it! :)

**Review by RasmusKjeldgaard — APPROVED** [review]
🎉 

⚠️ I believe this should have a UX review by Dennis before merge ⚠️

**Pull request merged**
